packages/stylelint-configs/eslint.config.mjs

```diff
@@ -0,0 +1,7 @@
+import workleapPlugin from "@workleap/eslint-plugin";
```

Comment:
FYI i think stylelint-configs can now be converted to an esm project since version 16

Reply:
I'd prefer to do that in a separate release if that's ok. Already a lot going on here.